Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/TestingSyncUpDetail.tutorial

```diff
@@ -68,6 +68,9 @@
       
       It is also possible to shorten this test quite a bit by using a non-exhaustive test store, as
       we did in <doc:PresentingSyncUpForm>, but we will leave that as an exercise for the reader.
+
+      As for the delete flow, since it involves its parent feature and some navigation with the dismiss 
+      helper, we'll test it later in the <doc:TestingNavigation>.
```

Comment:
📓 I've mention and link to the delete flow testing being now in the navigation testing chapter